### PR TITLE
Implement Functionality to Read and Write Schedule Data

### PR DESCRIPTION
**Pull request by alanlowzies**
This PR contains the functionality to read and write data related to the schedule feature. These data are written to the `schedule.txt` file, located in the `werkItResources` folder.

Some minor adjustments to some UI elements were also made, such as the listing of the plans (`plan /list`).

### CODE REVIEW

File: src/main/java/storage/FileManager.java [hunk 1]
```diff
@@ -232,9 +272,18 @@ public void checkAndCreateDirectoriesAndFiles() throws IOException {
...
+        if (!isWasScheduleFileAlreadyMade()) {
+            ui.printScheduleFileNotFoundMessage();
+            createScheduleFile();
+            ui.printScheduleFileCreatedMessage();
+        }
+
+        assert (Files.exists(getDirectoryPath())) : "Directory does not exist, but it should.";
+        assert (Files.exists(getExerciseFilePath())) : "Exercise file does not exist, but it should.";
+        assert (Files.exists(getWorkoutFilePath())) : "Workout file does not exist, but it should.";
+        // TODO: assert for plans (Haofeng?)
```
Comment (a1021492980) [inline src/main/java/storage/FileManager.java:285]: will do that, thanks for the message

**Review by a1021492980 — APPROVED** [review]
Good implementations, thank you.

File: src/main/java/storage/FileManager.java (hunk 1 above)
Comment (alanlowzies) [reply src/main/java/storage/FileManager.java:285]: > Good implementations, thank you.

Thank you :)

**Review by Musfirahe0556596 — APPROVED** [review]
Looks great!

**Comment by alanlowzies** [comment]
> Looks great!

Thank you :)

**Pull request merged**